app/Http/Middleware/RegularAndSuperAdminUser.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Suyabay\Http\Middleware;
+
+use Auth;
+use Closure;
+
+class RegularAndSuperAdminUser
```

Comment:
**Code Quality - Nitpik**
@andela-tolotin The Middleware class name here is confusing, please work on this